src/puppetlabs/puppetdb/query/edges.clj

```diff
@@ -23,14 +23,9 @@
 
 (pls/defn-validated munge-result-rows
   "Reassemble rows from the database into the final expected format."
-  [_
-   projected-fields :- [s/Keyword]
-   _
-   _]
+  [_ _ _]
```

Comment:
looks like this could just be changed to a `(def munge-result-rows (constantly (fn [rows] ....`